projects/plugins/jetpack/_inc/lib/class-jetpack-tweetstorm-helper.php

```diff
@@ -1325,7 +1342,11 @@ function ( $tweet ) {
 					$tweet['blocks'] = array();
 				} else {
 					// Remove the parts of the block data that the editor doesn't need.
-					$block_count = count( $tweet['blocks'] );
+					if ( is_countable( $tweet['blocks'] ) ) {
```

Comment:
Another case where it would be hard for it to ever not be a countable, since line 1341 has already verified that either `$tweet['blocks'][0]['attributes']` or `$tweet['blocks'][0]['clientId']` is set. So unless `$tweet['blocks']` is some kind of non-countable ArrayAccess or something else extremely weird...